### DISCUSSION TITLE
求助：我想把它作为一个网关使用，可以吗？

### DISCUSSION THREAD

**Discussion by wangbob0787** [docker-easyconnect/docker-easyconnect#191] [Q&A]
1 我在PVE虚拟机上安装了一个RouterOS（192.168.1.1）和一个Ubuntu Server（192.168.1.2）。
2 我想在Ubuntu Server 中采用docker的方式安装EasyConnect。
3 外网通过VPN连接RouterOS。
4 在RouterOS中将要访问EasyConnect的流量做路由标记，并将做了路由标记的流量指向Ubuntu Server（192.168.1.2，将它作为一个网关），再连接上EasyConnect（下一级网关）。

我的问题是：
1 是不是Ubuntu Server 需要开启网关服务？
2 Ubuntu Server 内是否按照您的说明（如下）就可以了？

<img width="1028" alt="截屏2022-11-13 21 06 20" src="https://user-images.githubusercontent.com/118112427/201523152-bc910e23-3070-454f-973d-51041290d130.png">

**Comment by Hagb** [comment]
>  1 是不是Ubuntu Server 需要开启网关服务？

网关服务其实是通过 Linux 内置的转发（forward）功能来做。系统防火墙的规则得允许这个转发

>  2 Ubuntu Server 内是否按照您的说明（如下）就可以了？

应该是，但是这个规则是让宿主机本地发出的流量走 EasyCnnect，而且和 NetworkManager 可能会有一些兼容性问题。这里需要根据你自己的实际情况来设置，如果需要把 Ubuntu Server 作为网关的话，`iif lo` 里的 lo 可以设为 Ubuntu Server 视角上的物理网卡名。

**Reply by ZHAIXINGZHAIYUE** [reply]
成功了吗
